<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminHomePage</class>
 <widget class="QWidget" name="adminHomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="rightMargin">
       <number>222</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_home_page">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 130, 53);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Home Page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_your_account">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 130, 53);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Your Account</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_manage_books">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 130, 53);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Manage Books</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 8pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Search by using at least one of the fields below and choose the book that interested in...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLineEdit" name="lineEdit_home_title">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(74, 74, 74);</string>
        </property>
        <property name="placeholderText">
         <string>Book Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_home_first_name">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(74, 74, 74);</string>
        </property>
        <property name="placeholderText">
         <string>Author's First Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_home_last_name">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(74, 74, 74);</string>
        </property>
        <property name="placeholderText">
         <string>Author's Last Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="actual_search_button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 130, 53);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>707</width>
         <height>341</height>
        </rect>
       </property>
       <widget class="QTableView" name="tableView_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>711</width>
          <height>151</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;
     border: 2px outset black;</string>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
       <widget class="QLineEdit" name="lineEdit_bookID_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>202</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(74, 74, 74);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>BookID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_pesel_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>202</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(74, 74, 74);</string>
        </property>
        <property name="placeholderText">
         <string>PESEL</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bookIt_button_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>240</y>
          <width>80</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 130, 53);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Book it!</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
